language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/document/BallerinaDocumentService.java

```diff
@@ -412,7 +412,7 @@ public void init(LanguageServer langServer,
             }
 
             try {
-                Optional<Project> project = this.workspaceManager.project(filePath.get());
+                Optional<Project> project = this.workspaceManagerProxy.get(fileUri).project(filePath.get());
```

Comment:
@nadeeshaan Should we use `workspaceManagerProxy.get()` or `get(fileUri)` here? Other methods are using the `get()` method.

Reply:
true... we don't need the fileURI as this won't be access during the cloned project scenarios ... will fix this.

Reply:
fixed